src/liballoc/arc.rs

```diff
@@ -200,16 +238,13 @@ impl<T: ?Sized> Arc<T> {
     /// # Examples
     ///
     /// ```
-    /// #![feature(arc_weak)]
-    ///
     /// use std::sync::Arc;
     ///
     /// let five = Arc::new(5);
     ///
     /// let weak_five = five.downgrade();
     /// ```
-    #[unstable(feature = "arc_weak",
-               reason = "Weak pointers may not belong in this module.")]
+    #[stable(feature = "arc_weak", since = "1.4.0")]
     pub fn downgrade(&self) -> Weak<T> {
```

Comment:
I'm fine with either -- my argument was symmetry, but I could see the argument that downgrade is different.